pydy/tests/test_system.py

```diff
@@ -395,3 +397,27 @@ def test_integrate(self):
         sys = System(self.kane, times=times)
         with testing.assert_raises(NotImplementedError):
             sys.generate_ode_function(generator='made-up')
+
+
+class TestJointSystem():
+
+    def setup(self):
+        self.system = System.using_joints()
```

Comment:
This is interesting. I don't know if I've even seen this done in Python. I think the typical way to do this would to support an `__init__` method that took no arguments. Or subclass System to make a JointSystem which overrides the `__init__` method.